pocketimdb/movies/utils.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime
+
+seconds_min = 60
+seconds_hour = seconds_min * 60
+seconds_day = seconds_hour * 24
+seconds_month = seconds_day * 30
+seconds_year = seconds_day * 365
+
+def get_timestamp_relative_diff(timestamp):
+    
+    diff = int((datetime.now() - timestamp).total_seconds())
+
+    if (diff < seconds_min):
+        return get_timestamp_relative_diff_desc(diff, 1, 'second')
+
+    if (diff < seconds_hour):
+        return get_timestamp_relative_diff_desc(diff, seconds_min, 'minute')
+
+    if (diff < seconds_day):
+        return get_timestamp_relative_diff_desc(diff, seconds_hour, 'hour')
+
+    if (diff < seconds_month):
+        return get_timestamp_relative_diff_desc(diff, seconds_day, 'day')
+
+    if (diff < seconds_year):
+        return get_timestamp_relative_diff_desc(diff, seconds_month, 'month')
+
+    return get_timestamp_relative_diff_desc(diff, seconds_year, 'year')
```

Comment:
ovakve stvari ne treba da convert-ujes na BE i da saljes nazad na FE, ovo se sad vise bavi nacinom reprezenaticje kada je nesto kreirano, tako da ovo moze zavisit od hiljadu klijneata od toga kako hoce ko da mu se prikaze, tako da uvijek podatke za BE saljes `raw`, cisto, a onda na FE radis nacin prikaza toga, da li ce to biti prije koliko je kreirano ili tacan datum ali u drugacijoj verziji itd... Jer ukoliko se bude promjenila zelja na koji nacin treba da bude prikazan format datum onda moras  implemrtnaciju na BE mijenjati umjesto samo na FE, a pogovo ako iams dvije FE apliakcije i razlicito prikazuju, tek tad ces imati problem sa ovim pristupom

Reply:
Mnogo sam razmisljao o ovome i eto opet pogresno pristupio.. Sustinski sam ovakve stvari maltene uvek radio na FE, ali sada sam ipak ovako uradio jer gledam da sto vise logike obavlja BE.. ali da, ocigledno je da ne treba da se bavi i reprezentacijom podataka, tako da prebacujem ovo na FE.